src/main/resources/languages/french.xml

```diff
@@ -2763,4 +2763,45 @@
         <value>Faire un don PayPal</value>
         <hint>About Menu</hint>
     </string>
+    <string>
+        <key>PaperSettings.Title</key>
+        <value>Paramètres papier</value>
+        <hint>Paper settings panel title</hint>
+    </string>
+    <string>
+        <key>PlotterSettingsPanel.Title</key>
+        <value>Paramètres du traceur</value>
+        <hint>panel title</hint>
+    </string>
+
+    
+    <string>
+        <key>TurtleRenderFactory.DEFAULT</key>
+        <value>Défaut</value>
+        <hint></hint>
+    </string>
+    <string>
+        <key>TurtleRenderFactory.BARBER_POLE</key>
+        <value>Poteau de barbier</value>
```

Comment:
The traduction is funny ;-)

Reply:
I'm struggling to find a translation that makes sense as it's a cultural idiom...
I will try something else ...